lib/phoenix_live_view/channel.ex

```diff
@@ -1308,4 +1322,18 @@ defmodule Phoenix.LiveView.Channel do
       _ -> nil
     end
   end
+
+  defp maybe_subscribe_to_live_reload({:noreply, state}) do
+    live_reload_config = state.socket.endpoint.config(:live_reload)
+
+    for {topic, _patterns} <- live_reload_config[:notify] || [] do
```

Comment:
Do we want to get only the `:live_view` topic perhaps?

```suggestion
    for {topic, _patterns} <- live_reload_config[:notify][:live_view] || [] do
```

Reply:
`live_reload_config[:notify]` is what we want I think because we made the `:notify` config on live reload library agnostic, so it broadcasts a message to those topics and any library can receive them.

Reply:
sorry, José is correct in this case. LiveView should only look for the "liveview" topic. Others would be for other libs to consume.

Reply:
Most likely “live_view” for consistency?